book/src/upgrading.md

```diff
@@ -0,0 +1,64 @@
+# Upgrading to new versions
+
+## From rinja v0.3 to askama v0.13
```

Comment:
We'll need to add a `from askama v0.12 to askama v0.13` too, a lot of things changed. A part of the list is present in the first blog post I wrote. It's important for binary operators.

Reply:
I added a "From askama v0.12 to askama v0.13" section. I wanted to keep this file short, only listing breaking changes. I made that more clear in the introduction of the page.